packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -542,7 +543,9 @@ describe('Transfer tests', function(this: any) {
         describe('Transfer CeloDollars', () => {
           describe('gasCurrency = CeloDollars >', () => {
             testTransferToken({
-              expectedGas: 189456,
+              // TODO(nategraf): Initializing a stable token account costs 5k additional gas.
+              // The first time this transfer is done, which is for the full node, this will be seen.
+              expectedGas: syncMode === 'full' ? 190303 : 175303,
```

Comment:
5k or 15k?

Reply:
I should really leave math to the computers